### PR TITLE
add wasm32 as a rust-std to use as target

### PR DESCRIPTION
**Pull request by ruben-arts**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged) Not sure if this is needed as i only want to build the new source?
* [ ] Reset the build number to `0` (if the version changed) ^^
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ruben-arts** [comment]
closes #153

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ rust_std_extra:
 - arm-linux-androideabi
 - armv7-linux-androideabi
 - i686-linux-android
+- wasm32-unknown-unknown
```
Comment (isuruf) [inline .ci_support/linux_64_.yaml:22]: This file is auto-generates. Please do the same change in `recipe/conda_build_config.yaml`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -51,6 +51,9 @@ source:
   - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-x86_64-linux-android.tar.gz  # [linux and x86_64]
     sha256: 177e719436e1acf79a3fd3d4b465ebe0d67337606a4b2c53732c26165f3e1738  # [linux and x86_64]
     folder: rust-std  # [linux and x86_64]
+  - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-wasm32-unknown-unknown  # [linux and x86_64]
```
Comment (isuruf) [inline recipe/meta.yaml:54]:
```suggestion
  - url: https://static.rust-lang.org/dist/rust-std-{{ version }}-wasm32-unknown-unknown.tar.gz  # [linux and x86_64]
```

File: .ci_support/linux_64_.yaml (hunk 1 above)
Comment (ruben-arts) [reply .ci_support/linux_64_.yaml]: Ah thanks! never knew that, just found the explanation in the readme!

**Comment by ruben-arts** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_.yaml (hunk 1 above)
Comment (pkgw) [reply .ci_support/linux_64_.yaml:22]: Sorry, I gave some inaccurate advice about this in #153 .

Comment (ruben-arts) [reply .ci_support/linux_64_.yaml:22]: @pkgw No problem, I learned from it so it had value!

**Pull request merged**